README.md

```diff
@@ -0,0 +1,11 @@
+# Mobile Book Club
+
+Mobile Book Club is a initiative where Nubank's Mobile Chapter gathers together once a week to discuss one (or more) chapter(s) of a book.
+
+This repo contains our notes and the code written during this meetings.
+
+## Current book
+
+### Data Structures and Algorithms in Swift
+- [Our Notes](https://github.com/nubank/mobile-book-club/tree/master/data-structures-and-algorithms-in-swift)
+- [Where buy the book](https://store.raywenderlich.com/products/data-structures-and-algorithms-in-swift)
```

Comment:
I think it is `Where to buy the book`, isn't it?

Reply:
```suggestion
- [Where to buy the book](https://store.raywenderlich.com/products/data-structures-and-algorithms-in-swift)
```